templates/blog/post_detail.html

```diff
@@ -0,0 +1,30 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <div class="post">
+    <h2 class="text-center">{{ post.title }}</h2>
+    <hr style="height: 2px; background-color: #9D7753">
+    <p style="font-size: large">{{ post.content }}</p>
+    <br>
+    <br>
+    <p>{{ post.commentaries.all.count }} Comment{{ post.commentaries.all.count|pluralize }}:</p>
```

Comment:
This line is a bit complex and hard to read. Consider creating a custom template filter or a method in your Post model to get the count of comments and its pluralized string. It would make your code cleaner and easier to understand.